tests/framework/extensions/clusters/clusters.go

```diff
@@ -296,3 +345,58 @@ func CreateRKE2Cluster(client *rancher.Client, rke2Cluster *provisioning.Cluster
 
 	return cluster, nil
 }
+
+// CreateK3SCluster is a "helper" functions that takes a rancher client, and the k3s cluster config as parameters. This function
+// registers a delete cluster fuction with a wait.WatchWait to ensure the cluster is removed cleanly.
+func CreateK3SCluster(client *rancher.Client, k3sCluster *provisioning.Cluster) (*provisioning.Cluster, error) {
+	cluster, err := client.Provisioning.Cluster.Create(k3sCluster)
```

Comment:
Same comment as above how different is k3s vs rke2 in terms of functions?

Reply:
I think in this case we should call ie CreateK3SRKE2Cluster, since this is exactly the same as its RKE2 counter part

Reply:
If we want, we can just use the CreateRKE2Cluster function. I only made this new function as to keep things specific to K3s, but not really an issue to use RKE2Cluster

Reply:
I would prefer that. We could even just name it CreateK3SRKE2Cluster